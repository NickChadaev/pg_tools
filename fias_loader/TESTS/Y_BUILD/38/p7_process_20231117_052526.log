2023-11-17 05:25:26 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_38 postgres /home/n.chadaev@abrr.local/Y_BUILD/38/stage_gar_c_38.csv 2023-11-13
--------------------------------------------------------
[05:25:26] ... Иркутская обл., id_region: 37, Execute: /home/n.chadaev@abrr.local/Y_BUILD/38/stage_gar_c_38.csv, process_id:25345
[05:25:26] ... Start process
[05:25:26] ...  -- Version
[05:25:26] ...  -- Очистка от данных
[05:25:26] ...  -- Убираем индексы
[05:25:26] ...  -- Set UNLOGGED
[05:25:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[05:25:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[05:25:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[05:25:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[05:25:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[05:25:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[05:25:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[05:25:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[05:25:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[05:25:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[05:25:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_REESTR_OBJECTS_20231113_85674f45-916a-4738-ab58-0a5f71cef3ca.XML
[05:46:04] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_ADDR_OBJ_20231113_57bb8039-17fd-45b7-baed-754c390a5f08.XML
[05:46:41] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_HOUSES_20231113_4f32b571-9ad2-4039-8d09-b5874a484512.XML
[06:03:31] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_STEADS_20231113_2e58722a-76e8-4f03-bbc2-36163753cfd2.XML
[06:06:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_APARTMENTS_20231113_d654375d-a63a-479c-9c6e-b9ced808447a.XML
[06:17:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_ROOMS_20231113_53d70cd2-f0e6-49a8-9b28-a931cd084343.XML
[06:18:09] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_CARPLACES_20231113_5ce33920-de20-481a-92f9-14d9cd6e70ea.XML
[06:18:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_ADM_HIERARCHY_20231113_c4a28244-2f49-4cfb-9423-47321a3816aa.XML
[06:47:02] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_MUN_HIERARCHY_20231113_11d2bf2a-4dda-4ddf-83de-0924c2ef656d.XML
[07:14:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_CHANGE_HISTORY_20231113_cff9cb79-11eb-45e2-8d6a-8e98c4d793f9.XML
[08:16:04] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_ADDR_OBJ_DIVISION_20231113_442ba97f-50a1-4ce3-9e5a-497be781fcc7.XML
[08:16:04] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_ADDR_OBJ_PARAMS_20231113_860861e1-8a62-4ca3-9a40-9deb9a906977.XML
[08:22:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_STEADS_PARAMS_20231113_a2742c5f-1ee9-474b-b329-20759fc23412.XML
[08:49:44] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_HOUSES_PARAMS_20231113_2d4f312d-45bd-4d5f-b8d5-5b2aa6e82c2e.XML
[10:18:37] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_APARTMENTS_PARAMS_20231113_93b426fb-f4c7-4740-a64d-0a0a19571eea.XML
[10:38:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_ROOMS_PARAMS_20231113_149bc7c3-b62c-48b2-b697-414635b41703.XML
[10:38:42] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_CARPLACES_PARAMS_20231113_67d6878f-2b86-4e6d-b190-6ecbfb535533.XML
[10:38:43] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/38/AS_NORMATIVE_DOCS_20231113_17ab3f49-9cdf-498d-a9e8-c7bbdf3fc86a.XML
[10:42:54] ...  -- Data finish
[10:42:54] ...  -- Set LOGGED
[10:43:38] ... Stop process
--------------------------------------------------------
