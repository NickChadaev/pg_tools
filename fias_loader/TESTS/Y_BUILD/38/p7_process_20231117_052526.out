 save_gar_version 
------------------
               81
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                81
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 1701345
gar_fias.as_addr_obj: 43070
gar_fias.as_houses: 1246053
gar_fias.as_steads: 215860
gar_fias.as_apartments: 945071
gar_fias.as_rooms: 16596
gar_fias.as_carplaces: 2628
gar_fias.as_adm_hierarchy: 2257759
gar_fias.as_mun_hierarchy: 2165106
gar_fias.as_change_history: 5095264
gar_fias.as_addr_obj_division: 477
gar_fias.as_addr_obj_params: 508675
gar_fias.as_steads_params: 2279505
gar_fias.as_houses_params: 6738338
gar_fias.as_apartments_params: 1566984
gar_fias.as_rooms_params: 21448
gar_fias.as_carplaces_params: 525
gar_fias.as_normative_docs: 342582
